Engine/device/mousew32.cpp

```diff
@@ -306,7 +309,16 @@ int mgetbutton()
     return NONE;  // don't allow holding button down
 
   if (butis & 1)
+  {
     toret = LEFT;
+#if defined(MAC_VERSION)
+    // j Ctrl-left click should be right-click
+    if (IsKeyPressed(405) || IsKeyPressed(406))
```

Comment:
I think I understand why this is needed, but I believe we need to write TODO about making this configurable later.
Some games may script Ctrl+LMB for particular in-game action.